### PR TITLE
Adding an IODA converter for balloon sonde data in JSON

### LINKED ISSUE #1424: Add an IODA converter to use JSON formatted WindBorne Data
## Description
A python ioda converter is needed to convert the raw data from WindBorne Systems JSON format into IODA format. 
First for testing to see the quality of data and then to be able to use in near-real time Skylab operations. 

## Requirements
Raw data from the JSON files is converted to IODA format

## Dependencies
PR: https://github.com/JCSDA-internal/ioda-converters/pull/1425

### PR DESCRIPTION
**Pull request by smaticka**
## Description

A python ioda converter is needed to convert the raw data from WindBorne Systems JSON format into IODA format.
First for testing to see the quality of data and then to be able to use in near-real time Skylab operations.

## Issue(s) addressed
Resolves #1424 

## Checklist
- [x] write a python converter to convert JSON data to IODA format (.nc file) 
- [x] use the converted data for 2023 11 01 00H and background data to run in hofx
- [x] compare results to JEDI.

### CODE REVIEW

**Comment by BenjaminRuston** [comment]
@smaticka do you know how to fix these:

```
checking ./balloon_sonde_json2ioda.py
./balloon_sonde_json2ioda.py:177:9: E722 do not use bare 'except'
./balloon_sonde_json2ioda.py:179:101: E261 at least two spaces before inline comment
./balloon_sonde_json2ioda.py:180:13: E722 do not use bare 'except'
./balloon_sonde_json2ioda.py:181:21: E261 at least two spaces before inline comment
./balloon_sonde_json2ioda.py:181:47: W291 trailing whitespace
./balloon_sonde_json2ioda.py:182:1: W293 blank line contains whitespace
./balloon_sonde_json2ioda.py:186:9: E722 do not use bare 'except'
./balloon_sonde_json2ioda.py:188:112: E261 at least two spaces before inline comment
./balloon_sonde_json2ioda.py:189:13: E722 do not use bare 'except'
./balloon_sonde_json2ioda.py:190:21: E261 at least two spaces before inline comment
```

a simple thing to use is to raise a `NotImplementedError`:
```
            print(" ... Error: could not do something or missing something")
            raise NotImplementedError('need a error handling or specific item to continue this implementation')
```

**Comment by PatNichols** [comment]
@smaticka @BenjaminRuston This is passing tests. The ci is still broken for the repo.

**Review by PatNichols — APPROVED** [review]
Thanks @smaticka

File: src/conventional/balloon_sonde_json2ioda.py [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+
+
+"""
+Python code to ingest JSON WindBorne Data
+"""
+
+import pdb
```
Comment (BenjaminRuston) [inline src/conventional/balloon_sonde_json2ioda.py:15]:
```suggestion
```

usually do the import of `pdb` on demand leave it as bare as possible in pre-loaded libraries

**Review by BenjaminRuston — APPROVED** [review]
thanks @smaticka this is working for both single and multiple files, start here for first cut and we'll add the humidity conversion using provided tables

**Pull request merged**